uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1020.php

```diff
@@ -0,0 +1,63 @@
+
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1020 {
+
+  /**
+   * Removes duplicated "Localise CiviHR" menu item
+   */
+  public function upgrade_1020() {
+    $this->up1020_deleteLocaliseCiviHRMenuItem();
+    $this->up1020_createLocaliseCiviCRMMenuItemIfItDoesNotExist();
+
+    return TRUE;
+  }
+
+  private function up1020_deleteLocaliseCiviHRMenuItem() {
+    civicrm_api3('Navigation', 'get', [
+      'sequential' => 1,
+      'label' => "Localise CiviHR",
+      'api.Navigation.delete' => ['id' => "\$value.id"],
+    ]);
+  }
+
+  private function up1020_createLocaliseCiviCRMMenuItemIfItDoesNotExist() {
+    $domain = CRM_Core_Config::domainID();
+    $params = ['return' => 'id', 'name' => 'Administer', 'domain_id' => $domain];
+    $administerId = (int) civicrm_api3('Navigation', 'getvalue', $params);
```

Comment:
I know the 'Administer' menu should always be there, but I guess it's possible that someone deleted it. Do you think we should we allow for this?

Reply:
Yeah it should always be there. I updated the upgrader to just not do anything if the menu is there, as, if that is the case, there isn't really much we can do.